src/neo/Ledger/VerifyResult.cs

```diff
@@ -47,6 +47,26 @@ public enum VerifyResult : byte
         /// </summary>
         PolicyFail,
 
+        /// <summary>
+        /// Indicates that the <see cref="Transaction"/> it's too big.
+        /// </summary>
+        TooBig,
```

Comment:
I would suggest using `TransactionOverSize` or something similar to replace `TooBig`